ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/scs2/SCS2AvatarSimulationFactory.java

```diff
@@ -332,10 +334,35 @@ private void setupYoVariableServer()
       }
    }
 
+   private SCS2JointDesiredOutputWriterFactory getDefaultOutputWriterFactory()
+   {
+      return (controllerInput, controllerOutput) ->
+      {
+         boolean writeBeforeEstimationTick = true;
+         if (kinematicsSimulation.get())
+            return new SCS2KinematicsSimulationOutputWriter(controllerInput, controllerOutput, writeBeforeEstimationTick);
+         else
+            return new SCS2OutputWriter(controllerInput, controllerOutput, writeBeforeEstimationTick);
+      };
```

Comment:
I added this since you last looked to make it easier to use the new feature. This just sets the default output writer appropriately, but the user is still free to set the outputwriter to something custom.